.github/workflows/ci.yaml

```diff
@@ -73,18 +73,17 @@ jobs:
       if: steps.cache.outputs.cache-hit != 'true'
       run: |
         # https://github.com/actions/virtual-environments/issues/5595
-        ANDROID_ROOT="/usr/local/lib/android"
-        ANDROID_SDK_ROOT="${ANDROID_ROOT}/sdk"
         SDKMANAGER="${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager"
         echo "y" | $SDKMANAGER "ndk;21.4.7075529"
-        export ANDROID_NDK="${ANDROID_SDK_ROOT}/ndk-bundle"
-        ln -sfn $ANDROID_SDK_ROOT/ndk/21.4.7075529 $ANDROID_NDK
     - name: Install tox4j dependencies
       if: steps.cache.outputs.cache-hit != 'true'
       run: sudo apt-get update && sudo apt install yasm
     - name: Build tox4j
       if: steps.cache.outputs.cache-hit != 'true'
       run: |
+        ANDROID_ROOT="/usr/local/lib/android"
+        ANDROID_SDK_ROOT="${ANDROID_ROOT}/sdk"
```

Comment:
Do we even need to set these two? I just grabbed them from the linked issue, and it looks like now you're calling sdkmanager using `ANDROID_SDK_ROOT` without setting it first.

Reply:
Oops. Yeah I left them for readability but I will inline them all now.

Reply:
No, I mean it looks like ANDROID_SDK_ROOT is already set to a sane value since we're calling sdkmanager via it above. :P
```
SDKMANAGER="${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager"
echo "y" | $SDKMANAGER "ndk;21.4.7075529"
```

Reply:
Oh. Interesting

Reply:
Yeap: https://gist.github.com/sonots/070f1059b71dd2dec8e43ec513c3f8f0#file-runs-on-ubuntu-latest-txt-L8